### ISSUE TITLE
[Bug]: using vllm-ascend 0.9.2rc1, I failed to find the corresponding torch-npu 2.5.1.post1.dev20250619 during software traceability

### ISSUE THREAD

**Issue by amuletYY** [vllm-project/vllm-ascend#2624]
### Your current environment

<details>
<summary>

I am creating a compliance image and need to trace the software image, but torch-npu 2.5.1.post1.dev20250619 cannot be found.
According to the official help documentation, have chosen a higher release version of torch npu ---- torch npu 2.7.1rc1. 
But still not match, the log shows that only torch npu 2.5.1.post1.dev20250619 can be selected.

the official help documentation is as flows:
<html>
<body>
<h2 id="prerequisites" md-src-pos="2977..2993" style="box-sizing: border-box; position: relative; margin-top: 1em; margin-bottom: 16px; font-weight: bold; line-height: 1.2; font-size: 1.8em; padding-top: 0.6em; color: rgb(188, 190, 196); font-family: Helvetica, Arial, freesans, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(30, 31, 34); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Prerequisites</h2><ul md-src-pos="2995..3235" style="box-sizing: border-box; padding: 0px 0px 0px 2em; margin-top: 16px; margin-bottom: 16px; color: rgb(188, 190, 196); font-family: Helvetica, Arial, freesans, sans-serif; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(30, 31, 34); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><li md-src-pos="2995..3063" style="box-sizing: border-box;">Hardware: Atlas 800I A2 Inference series, Atlas A2 Training series</li><li md-src-pos="3064..3075" style="box-sizing: border-box;">OS: Linux</li><li md-src-pos="3076..3235" style="box-sizing: border-box;">Software:<ul md-src-pos="3090..3235" style="box-sizing: border-box; padding: 0px 0px 0px 2em; margin-top: 0px; margin-bottom: 0px; color: rgb(188, 190, 196);"><li md-src-pos="3090..3113" style="box-sizing: border-box;">Python &gt;= 3.9, &lt; 3.12</li><li md-src-pos="3116..3133" style="box-sizing: border-box;">CANN &gt;= 8.2.rc1</li><li md-src-pos="3136..3192" style="box-sizing: border-box;">PyTorch &gt;= 2.5.1, torch-npu &gt;= 2.5.1.post1.dev20250619</li><li md-src-pos="3195..3235" style="box-sizing: border-box;">vLLM (the same version as vllm-ascend)</li></ul></li></ul><h2 id="getting-started" md-src-pos="3237..3255" style="box-sizing: border-box; position: relative; margin-top: 1em; margin-bottom: 16px; font-weight: bold; line-height: 1.2; font-size: 1.8em; padding-top: 0.6em; color: rgb(188, 190, 196); font-family: Helvetica, Arial, freesans, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(30, 31, 34); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Getting Started</h2><p md-src-pos="3257..3326" style="box-sizing: border-box; margin-top: 16px; margin-bottom: 16px; color: rgb(188, 190, 196); font-family: Helvetica, Arial, freesans, sans-serif; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(30, 31, 34); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><span md-src-pos="3257..3325" style="box-sizing: border-box;">Please use the following recommended versions to get started quickly</span>:</p>
Version | Release type | Doc
-- | -- | --
v0.9.2rc1 | Latest release candidate | QuickStart and Installation for more details
v0.7.3.post1 | Latest stable version | QuickStart and Installation for more details

</body>
</html>

</body>
</html>
</summary>

```text
Your output of above commands here
```

</details>
Getting Started
Please use the following recommended versions to get started quickly:

Version	Release type	Doc
v0.9.2rc1	Latest release candidate	[QuickStart](https://vllm-ascend.readthedocs.io/en/latest/quick_start.html) and [Installation](https://vllm-ascend.readthedocs.io/en/latest/installation.html) for more details
v0.7.3.post1	Latest stable version	[QuickStart](https://vllm-ascend.readthedocs.io/en/stable/quick_start.html) and [Installation](https://vllm-ascend.readthedocs.io/en/stable/installation.html) for more detailsPrerequisites
Hardware: Atlas 800I A2 Inference series, Atlas A2 Training series
OS: Linux
Software:
Python >= 3.9, < 3.12
CANN >= 8.2.rc1
PyTorch >= 2.5.1, torch-npu >= 2.5.1.post1.dev20250619
vLLM (the same version as vllm-ascend)
Getting Started
Please use the following recommended versions to get started quickly:

Version	Release type	Doc
v0.9.2rc1	Latest release candidate	[QuickStart](https://vllm-ascend.readthedocs.io/en/latest/quick_start.html) and [Installation](https://vllm-ascend.readthedocs.io/en/latest/installation.html) for more details
v0.7.3.post1	Latest stable version	[QuickStart](https://vllm-ascend.readthedocs.io/en/stable/quick_start.html) and [Installation](https://vllm-ascend.readthedocs.io/en/stable/installation.html) for more details

### 🐛 Describe the bug

Successfully installed torch-npu-2.7.1rc1
WARNING: Running pip as the "root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/
warnings/venv
+ pip install vllm-ascend==0.9.2rc1
WARNING: When the --extra-index-url option is used, the installation package is downloaded from an unverified URL. Exercise caution when using this option.
WARNING: Running pip install with root privileges is generally not a good idea. Try pip install --user’ instead.
Looking in indexes: ...
Collecting vllm-ascend=-0.9.2rc1
Downloading 
...
ERROR: Could not find a version that satisfies the requirement torch-npu==2.5.1.post1.dev20250619 (from vllm-ascend) (from versions: 1.11.0.post10, 2.1.Orc1, 2.1.0, 2.1.0.post3, 2.1.0.post6, 2.1.0.post7, 2.1.
0.post8, 2.1.0.post10, 2.1.0.post12, 2.1.0.post13, 2.2.0rc1, 2.2.0, 2.2.0.post2, 2.3.1rc1, 2.3.1, 2.3.1.post2, 2.3.1.post4, 2.3.1.post6, 2.4.0rc1, 2.4.0, 2.4.0.post2, 2.4.0.post4, 2.5.1rc1, 2.5.1, 2.5.1.post1
, 2.6.0rc1, 2.6.0, 2.7.1rc1, 2.8.0rc1)
ERROR: No matching distribution found for torch-npu==2.5.1.post1.dev20250619

**Comment by leo-pony** [comment]
@amuletYY 
v0.9.2rc1 is a transition version, you can try with [v0.10.1rc1](https://github.com/vllm-project/vllm-ascend/releases/tag/v0.10.1rc1).
Notice: For dev torch-npu version make sure following step been executed.
https://vllm-ascend.readthedocs.io/en/latest/installation.html
<img width="907" height="163" alt="Image" src="https://github.com/user-attachments/assets/cfe92bf8-3c75-43ce-9c9b-e476346d7580" />
